server/api/batches.py

```diff
@@ -82,9 +82,33 @@ def serialize_batch(batch: Batch) -> JSONDict:
             }
             for tally_sheet in batch.result_tally_sheets
         ],
+        "lastEditedBy": construct_batch_last_edited_by_string(batch),
     }
 
 
+def construct_batch_last_edited_by_string(batch: Batch) -> Optional[str]:
+    if batch.last_edited_by_support_user_email:
+        return batch.last_edited_by_support_user_email
+    if batch.last_edited_by_user_id:
+        user = batch.last_edited_by_user
+        assert (
+            user is not None
+        ), f"Unable to find user with ID {batch.last_edited_by_user_id}"
+        return user.email
```

Comment:
I think you can simplify this without hurting performance due to the joinedload
```suggestion
    if batch.last_edited_by_user:
        return batch.last_edited_by_user.email
```